expyfun/stimuli/_tracker.py

```diff
@@ -0,0 +1,837 @@
+                                    1 - self._chance)
+        self._max_p_val = binom.cdf(self._n_wrong + (self._max_trials -
+                                                     self._n_trials),
+                                    self._max_trials, 1 - self._chance)
+        if ((self._p_val <= self._alpha) or
+                (self._min_p_val >= self._alpha and self._stop_early)):
+            if self._n_trials >= self._min_trials:
+                self._stopped = True
+        if self._n_trials == self._max_trials:
+            self._stopped = True
+
+    # =========================================================================
+    # Define all the public properties
+    # =========================================================================
+    @property
+    def alpha(self):
+        return self._alpha
+
+    @property
+    def chance(self):
+        return self._chance
+
+    @property
+    def max_trials(self):
+        return self._max_trials
+
+    @property
+    def stop_early(self):
+        return self._stop_early
+
+    @property
+    def p_val(self):
+        return self._p_val
+
+    @property
+    def min_p_val(self):
+        return self._min_p_val
+
+    @property
+    def max_p_val(self):
+        return self._max_p_val
+
+    @property
+    def n_trials(self):
+        return self._n_trials
+
+    @property
+    def n_wrong(self):
+        """The number of incorrect trials so far
+        """
+        return self._n_wrong
+
+    @property
+    def n_correct(self):
+        """The number of correct trials so far
+        """
+        return self._n_correct
+
+    @property
+    def pc(self):
+        """Proportion correct (0-1, NaN before any responses made)
+        """
+        return self._pc
+
+    @property
+    def responses(self):
+        """The response history
+        """
+        return self._responses
+
+    @property
+    def stopped(self):
+        """Is the tracker stopped
+        """
+        return self._stopped
+
+    @property
+    def success(self):
+        """Has the p-value reached significance
+        """
+        return self._p_val <= self._alpha
+
+    @property
+    def x_current(self):
+        """Included only for compatibility with TrackerDealer
+        """
+        return self._x_current
+
+    @property
+    def x(self):
+        """Included only for compatibility with TrackerDealer
+        """
+        return np.array([self._x_current for _ in range(self._n_trials)])
+
+    @property
+    def stop_rule(self):
+        return 'trials'
+
+
+# =============================================================================
+# Define a container for interleaving several tracks simultaneously
+# =============================================================================
+# TODO: Make it so you can add a list of values for each dimension (such as the
+# phase in a BMLD task) and have it return that
+
+# TODO: eventually, make a BaseTracker class so that TrackerDealer can make
```

Comment:
What sayest thou, @Eric89GXL? I should be totally honest: these aren't things I'm likely to address in the near future, so the issues will likely sit there a while.